pmd-ui/src/main/java/net/sourceforge/pmd/util/fxdesigner/model/XPathSuggestions.java

```diff
@@ -0,0 +1,98 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.util.fxdesigner.model;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Enumeration;
+import java.util.List;
+
+
+
+
+public class XPathSuggestions {
+    private List<String> xPathSuggestions = new ArrayList<>();
+    private String packagename;
+    private String language;
+
+    public XPathSuggestions(String language) {
```

Comment:
It would be better to pass a Language instance and let the constructor get the string name it needs.